make/Makefile.ui.mk

```
@@ -19,7 +19,7 @@ yarn-start:
 
 ## cypress-run: Runs all the cypress frontend integration tests locally without the GUI (i.e. headless).
 cypress-run:
-	@cd ${ROOTDIR}/frontend && yarn cypress:run --headless --env CYPRESS_NUM_TESTS_KEPT_IN_MEMORY=0 --env CYPRESS_VIDEO=false
+	@cd ${ROOTDIR}/frontend && yarn cypress:run --headless --env CYPRESS_NUM_TESTS_KEPT_IN_MEMORY=0 --config video=false
```

:+1: while you're at it, could you update `CYPRESS_NUM_TESTS_KEPT_IN_MEMORY` too?

```suggestion
	@cd ${ROOTDIR}/frontend && yarn cypress:run --headless --config numTestsKeptInMemory=0,video=false
```